test/main.sh

```diff
@@ -99,7 +102,7 @@ spawn_lxd() {
   # overwrites the environment and we would lose LXD_DIR's value otherwise.
   local LXD_DIR
   echo "==> Spawning lxd on $1 in $2"
-  LXD_DIR=$2 lxd $debug --tcp $1 &
+  LXD_DIR=$2 lxd $debug --tcp $1 > $2/lxd.log &
```

Comment:
On Tue, Apr 21, 2015 at 07:54:27PM -0700, Stéphane Graber wrote:
> > @@ -99,7 +102,7 @@ spawn_lxd() {
> >    # overwrites the environment and we would lose LXD_DIR's value otherwise.
> >    local LXD_DIR
> >    echo "==> Spawning lxd on $1 in $2"
> > -  LXD_DIR=$2 lxd $debug --tcp $1 &
> > +  LXD_DIR=$2 lxd $debug --tcp $1 > $2/lxd.log &
> 
> Getting live output is pretty valuable. Can you instead do:
> 
>     2>&1 | tee $2/lxd.log &

Agreed, though I considered that more important for the lxc
concurrent commands (which i haven't yet pushed) - i'll change
it tomorrow.